bears/rest/reSTLintBear.py

```diff
@@ -14,13 +28,17 @@ class reSTLintBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
 
-    def run(self, filename, file):
+    def run(self, filename, file, ignore_unknown_roles: bool=False):
```

Comment:
gah I am not sure if we deprecate the bear, it will be worth doing this.

But now that I have seen `rstcheck's` way of doing this, I would also like to have two params `ignore_directives`, `ignore_roles`, which takes in a list of directives/roles to ignore. And then obviously do the stuff with regex and the helper function.